library/dicom_frame.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require_once('../interface/globals.php');
+
+$web_path = $_REQUEST['web_path'];
+$patid = $_REQUEST['patient_id'];
+$docid = $_REQUEST['document_id'];
+$web_path .= '&retrieve&patient_id=' . $patid . '&document_id=' . $docid . '&as_file=false'
+?>
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Throw a comment here that this is code is not compatible with bootstrap (to avoid us trying to convert this to bootstrap in the future :) ).